.eslintrc

```diff
@@ -15,17 +15,27 @@
     "jsx": true
   },
   "rules": {
-    "quotes": 0,
     "no-underscore-dangle": 0,
     "curly": [2, "multi-line"],
     "camelcase": [2, {"properties": "never"}],
-    "space-in-parens": [2, "never"],
+    "space-in-parens": [2, "always"],
```

Comment:
I would argue that neither team have come up with a reason why either is good or important. Therefore i think we should potentially leave this out of linting rules.